manual/en/docs/flow/actions/contains_string_action.md

```diff
@@ -0,0 +1,18 @@
+# ContainsString Plugin
+
+This plugin checks if data field contains defined prefix.
```

Comment:
prefix is only the leading part of the string. I would rename it to "substring". "prefix" is a bit misleading and confusing